WORKSPACE

```diff
@@ -13,22 +13,22 @@
 # limitations under the License.
 
 workspace(name = "build_bazel_rules_typescript")
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
-http_archive(
+# TEMPORARY: grab rules_nodejs from Alex fork
```

Comment:
Needs to be changed after https://github.com/bazelbuild/rules_nodejs/pull/279 is merged

Reply:
done